R/load_data_vegetation.R

```diff
@@ -113,7 +123,19 @@ load_data_vegetation <-
           is.na(.data$plotarea_ha),
           .data$totalplotarea_ha,
           .data$plotarea_ha
-        )
+        ),
+      # correct for NA instead of < 1% (waterlayer and soildisturbance only)
+      total_waterlayer_cover_id =
+        ifelse(is.na(.data$total_waterlayer_cover_id) &
+                 .data$not_na_waterlayer_cover,
+               1,
+               total_waterlayer_cover_id),
+      total_soildisturbance_game_id =
+        ifelse(is.na(.data$total_soildisturbance_game_id) &
+                 .data$not_na_soildisturbance_game,
+               1,
+               total_soildisturbance_game_id),
+
```

Comment:
```suggestion
               total_soildisturbance_game_id)
```
En dan de komma nog even weghalen.  ;-)